packages/core/admin/ee/admin/content-manager/pages/EditView/InformationBox/components/AssigneeSelect/AssigneeSelect.js

```diff
@@ -22,18 +24,30 @@ export function AssigneeSelect() {
     isSingleType,
     onChange,
   } = useCMEditViewDataManager();
+  const permissions = useSelector(selectAdminPermissions);
   const { formatMessage } = useIntl();
   const { formatAPIError } = useAPIErrorHandler();
   const toggleNotification = useNotification();
   const { put } = useFetchClient();
-  const { users, isLoading, isError } = useAdminUsers();
+  const {
+    allowedActions: { canReadUsers },
+  } = useRBAC({
+    ...permissions.settings.auditLogs,
+    readUsers: permissions.settings.users.read,
+  });
+  const { users, isLoading, isError } = useAdminUsers(
+    {},
+    {
+      enabled: canReadUsers,
+    }
+  );
 
   const currentAssignee = initialData?.[ASSIGNEE_ATTRIBUTE_NAME] ?? null;
 
   const handleChange = async ({ value: assigneeId }) => {
     mutation.mutate({
       entityId: initialData.id,
-      assigneeId,
+      assigneeId: parseInt(assigneeId, 10),
```

Comment:
i've seen some people change `parseInt` to use `Number` do you know the pros/cons? 🤔

Reply:
I don't but I find `parseInt` more appropriate because it needs to be an integer instead of any Number. What do you think?

Reply:
ah i think the difference is `Number` will take anything, meanwhile `parseInt` only takes strings.

I prefer the latter too.